src/modules/launcher/Wox.Infrastructure/Image/ImageHashGenerator.cs

```diff
@@ -38,8 +42,9 @@ public string GetHashFromImage(ImageSource imageSource)
                     }
                 }
             }
-            catch
+            catch (System.Exception e)
```

Comment:
Unfortunately, using `Exception` instead of `System.Exception` gives me a compiler error since it clashes with `Wox.Infrastructure.Exception`
`CS0118	'Exception' is a namespace but is used like a type	`

Reply:
Sounds good. thanks!